xla/service/gpu/cublas_cudnn.h

```diff
@@ -181,37 +181,22 @@ bool IsCustomCallToDnnNorm(const HloInstruction& hlo);
 
 // The fused_mha_rewriter phase where each of the MHA signatures are pattern
 // matched and rewritten into a custom-call with specific custom-call target.
-// The custom-call target specifies the MHA signature. For example,  BMM1 - Bias
-// - Scale - Mask - Softmax - BMM2 pattern can have the target as
-// cudnn$fmhaBiasScaleMaskSoftmax.
+// The custom-call target specifies the MHA signature. For example,  BMM1 -
+// Scale
```

Comment:
nit: please reflow the comment such that the following line is moved to this line (until 80 character limit)

Reply:
fixed